Main.java

```diff
@@ -9,7 +10,31 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    // detirmine the greater number
+
+    // Create a scanner for user input
+    Scanner input = new Scanner(System.in);
+    
+    // ask user to enter two numbers
+    System.out.print("Please enter a number:");
+    int number1 = input.nextInt();
+
+    System.out.print("Please enter another number:");
+    int number2 = input.nextInt();
+
+    // determine which number is greater    
+    if(number1>number2){
+      // tell them if the number is greater
+      System.out.print(number1 + "is greater" );
+    }else if(number1==number2){
+      // tell them the numbers are equal
+      System.out.print(number1 + "and" + number2 + "are equal");
```

Comment:
Make sure you are trying to match the sample output as closely as you can. IT does say in the question to say "There is no bigger number".